resources/asciidoctor/lib/chunker/breadcrumbs.rb

```diff
@@ -12,9 +12,65 @@ def generate_breadcrumbs(doc, section)
       result = ['<div class="breadcrumbs">']
       result += generate_breadcrumb_links(section).reverse
       result << '</div>'
+      if result[2].to_s.include? 'APM'
+        result.insert(2, generate_apm_breadcrumbs)
+      end
+      if result[2].to_s.include? 'ECS Logging'
+        result.insert(2, generate_ecslogging_breadcrumbs)
+      end
```

Comment:
The third element in the `result` array is always the title of the selected book. These `if` statements check to see if certain text exists, and if it does, inserts the relevant dropdown HTML into the `result` array.